### PR TITLE
Record extra build info required for assessment storage

### PR DESCRIPTION
**Pull request by glenjamin**

### CODE REVIEW

File: test/unit/server/routes/health.spec.js [hunk 1]
```diff
@@ -12,14 +12,15 @@ describe('GET /health', () => {
     app.use('/health', healthEndpoint);
   });
 
-  it('responds with { status: "OK" }', (done) => {
-    request(app)
+  it('responds with { status: "OK" }',
+    () => request(app)
```
Comment (romidane) [inline test/unit/server/routes/health.spec.js]: interesting the done is no longer required?

Comment (romidane) [inline test/unit/server/routes/health.spec.js]: Is it because you're returning a promise

File: test/unit/server/routes/health.spec.js [hunk 2]
```diff
@@ -33,18 +34,16 @@ describe('GET /health', () => {
...
-        gitRef: 'bar',
-      }, done);
-    });
+    it('responds with the build information inside the status ended',
+      () => request(app)
+        .get('/health')
+        .expect('Content-Type', /json/)
+        .expect(200)
+        .expect((res) => {
+          expect(res.body).to.have.property('status', 'OK');
```
Comment (romidane) [inline test/unit/server/routes/health.spec.js]: Good idea 👍

**Comment by romidane** [comment]
Could you upate the README with the new health endpoint JSON?

File: test/unit/server/routes/health.spec.js (hunk 1 above)
Comment (glenjamin) [inline test/unit/server/routes/health.spec.js]: yeah, promises.

**Comment by glenjamin** [comment]
@romidane I've updated, but now the "unit" test for the health endpoint is really an integration test of the build-info script + the endpoint. Is that ok or should it be somewhere else?

**Comment by romidane** [comment]
@glenjamin, I think you can extract the build-info script test into an different folder perhaps scripts and in the health endpoint test keep it the way it was but just add the new values that would have appeared in the dummy build-info file. That way the endpoint both scripts can remain unit tests?

**Comment by glenjamin** [comment]
@romidane I've split the tests, feels a bit neater now. Happy for me to merge?

**Comment by romidane** [comment]
@glenjamin Looks great 👍 :shipit:

**Pull request merged**